File: xindice/rhino/read/xindice_read.js
// ------------------------------------------------------------------
//
//	xindice_read.js
//
//						Jun/14/2011
//
// ------------------------------------------------------------------
load ('/var/www/data_base/common/rhino_common/xml_manipulate.js');
load ('/var/www/data_base/common/rhino_common/text_manipulate.js');

print	("*** 開始 ***");

var url_in="http://host_dbase:8888/xindice/db/cities/cities"


var xml_str  = readUrl(url_in);
// print (xml_str);
var dict_aa = xml_to_dict_proc (xml_str);

dict_display_proc (dict_aa);

print	("*** 終了 ***");

// ------------------------------------------------------------------

File: xindice/jruby/read/xindice_read.rb
#! /usr/bin/jruby
#
#	xindice_read.rb
#
#					May/19/2011
#
require "rexml/document"
require	"net/http"
load '/var/www/data_base/common/ruby_common/xml_manipulate.rb'
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
include REXML
#
# -----------------------------------------------------------------
print	"*** 開始 ***\n"
#
host= "host_dbase"
port = 8888
path="/xindice/db/cities/cities"
#
Net::HTTP.start(host,port)do |http|
		contents=http.get(path)
#		print contents.body
		dict_aa=xml_to_dict_proc(contents.body)
		dict_display_proc(dict_aa)
	end
#
print	"*** 終了 ***\n"
# -----------------------------------------------------------------
